genom.py

```diff
@@ -1,85 +1,98 @@
 from tools.dna_rna_tools import (valid, transcribe, reverse,
                                  complement, reverse_complement)
 from tools.filter_parameters import (ch_length_bounds, ch_gc_bounds,
-                                     ch_quality_threshold)
+                                     ch_quality_threshold, read_fastq,
+                                     write_fastq)
 
 
 def run_dna_rna_tools(*args) -> str | list:
     """
-    Функция run_dna_rna_tools() возвращает  последовательности нуклеиновых
-    кислот после проведения указанной в аргументе процедуры над переданными в
-    функцию изначальными последовательностями ДНК или РНК.
-    Функция проводит проверку на возможность существованияи значальных
-    последовательностей ДНК и РНК. При наличии Т и U в одно последовательности,
-    функция прекращается и возвращает сообщение пользователю:
+    Function run_dna_rna_tools() returns nuc acid sequence after actions
+    on initial RNA or DNA nucs sequence ordered by the procedure
+    wrote in argument.
+    Function make check for exsistance of initial nucs sequences, by checking
+    simultanously presence of T and U in seq.
+    In case of negarive resalt func returns:
     "Please insert correct sequence"
-    Вариант вызова функции:
+    Possible variant of func colling:
     run_dna_rna_tools('ATG','aT','reverse')
-        'ATG','aT',... - произвольное количество аргументов с
-        последовательностями ДНК или РНК в формате str.
-        'reverse' - всегда последний аргумент функции, с указанием процедуры
-        которую необходимо выполнить в формате str.
-        Список процедур:
-            'reverse' - вернуть развёрнутую последовательность.
-            'transcribe' - вернуть транскрибированную последовательность.
-            'complement' - вернуть комплементарную последовательность.
-            'reverse_complement' - вернуть обратную комплементарную.
-            последовательность.
-    Возвращаемое значение: строка с одной последовательностью нуклеиновых
-    кислот или список последовательностей нуклеиновых кислот, в зависимости
-    от количества переданных изначально послдеовательностей.
+        'ATG','aT',... - optional quantity of arguments, consisting of
+        nuc sequences in str format.
+        'reverse' - always last parameter in function, with istruction
+        for required procedure.
+        Procedure list:
+            'reverse' - return reversed sequence.
+            'transcribe' - return transcribed sequence.
+            'complement' - return comlemented sequence.
+            'reverse_complement' - returns reversed and complemented seq.
+    Return value: string with one sequence of nucs or list with multiple
+    seqs of nucs, depending on quantity of initial seqs.
     """
     seq: tuple = args[: -1]
-    if valid(seq) is False:  # Проверка введёной последовательности
+    if valid(seq) is False:
```

Comment:
лучше так
```suggestion
    if not valid(seq):
```
даже читается почти как английский текст!